trade-finance-manager-api/test-helpers/upsert-tfm-user-from-entra-id-response.ts

```diff
@@ -0,0 +1,3 @@
+import { aTfmUser } from '@ukef/dtfs2-common/mock-data-backend';
+
+export const anUpsertTfmUserFromEntraIdUserResponse = () => aTfmUser();
```

Comment:
Do we really need this - it's only used in one place and just returns another function.